packages/shared-ui/src/flowHandler/constants.ts

```diff
@@ -25,6 +25,12 @@ export const createLoginIdentifierType = (v: ApiLoginIdentifierType): LoginIdent
   }
 };
 
+export enum LoginIdentifierConfigType {
+  Email = 'email',
+  Phone = 'phone_number',
+  Username = 'custom',
```

Comment:
This is an outdated type, we don't use it anymore (the only valid login identifiers we have are: email, phone and username and I guess we already have a type for them in this package). 

Maybe we took the wrong type for loginIdentifiers when we implemented this new identifier-list-config endpoint => please check that.